scripts/get_latest_release.py

```diff
@@ -1,22 +1,32 @@
-#!/usr/local/bin/python
+#!/usr/bin/env python
 
 import json
 import requests
 import sys
 
+DEFAULT_VERSION = "1.0.0"
+
 HTTP_OK = 200
+
 # Try to get the latest release version
 r = requests.get("https://api.github.com/repos/contiv/install/releases/latest")
 if r.status_code != HTTP_OK:
    # Get all the releases (include pre-releases) and pick the first of them
    r = requests.get("https://api.github.com/repos/contiv/install/releases")
    if r.status_code != HTTP_OK:
       # Return a known version
-      print "1.0.0-beta.3"
+      print DEFAULT_VERSION
```

Comment:
I don't want to change the behavior of the existing script, though I do want to have a conversation about the "defaulting to some version" behavior.  Waiting for @neelimamukiri to get back from PTO to ask why it's currently that way